pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -346,21 +349,29 @@ func (k *Kubernetes) ConfigServicePorts(name string, service kobject.ServiceConf
 		targetPort.IntVal = port.ContainerPort
 		targetPort.StrVal = strconv.Itoa(int(port.ContainerPort))
 
+		// decide the name based on whether we saw this port before
+		name := strconv.Itoa(int(port.HostPort))
+		if _, ok := seenPorts[int(port.HostPort)]; ok {
+			// https://github.com/kubernetes/kubernetes/issues/2995
+			if service.ServiceType == string(api.ServiceTypeLoadBalancer) {
+				panic("Services of type LoadBalancer cannot use both TCP and UDP for same port " + name)
+			}
+			name = fmt.Sprintf("%s-%s", name, port.Protocol)
+		}
+		name = strings.ToLower(name)
```

Comment:
I think that the output should be uppercase rather than lowercase. Only reason being is that UDP and TCP is by-default uppercase within Kubernetes. Would make more sense for the names to be: `80-UDP` or `80-UDP` by default.